### DISCUSSION TITLE
Deployment Failed

### DISCUSSION THREAD

**Discussion by doverradio** [vercel/next.js#12886] [Help]
I tried following the deployment instructions and uploaded my project to gitlab (not github).  After, I connected my Vercel account to my Gitlab account.  Then I tried to deploy.  After waiting for 2 minutes, it said Deployment Failed.  It doesn't appear to give any reason why.  What am I to do?  I need to deploy this today.

**Comment by doverradio** [comment]
I'm sorry but being new to this I didn't see the build logs.  After
checking there, I saw this:

10:24:55.640
Failed to compile.
10:24:55.640
./pages/admin/crud/blogs.js
10:24:55.640
Module not found: Can't resolve '../../../components/crud/BlogRead' in
'/zeit/21037c54/pages/admin/crud'
10:24:55.640
> Build error occurred
10:24:55.641
Error: > Build failed because of webpack errors
10:24:55.641
at build (/zeit/21037c54/node_modules/next/dist/build/index.js:13:913)
10:24:55.666
error Command failed with exit code 1.
10:24:55.666
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this
command.
10:24:55.675
Error: Command "yarn run build" exited with 1

**Reply by etc-tiago** [reply]
The logs say that there is no **components/crud/BlogRead**

**Comment by doverradio** [comment]
Yes, I solved it by adding the missing files.  Now the site successfully
deployed!  Thank you!

I tried adding my domain and it said I needed to update the nameservers to
zeit's.  So I did that at my domain registrar name.com.  Just waiting now
for it to accept that I added those.
